File: 백준/Bronze/34529. Acquiring SW－IT Corn/Acquiring SW－IT Corn.c
#include <stdio.h>
int main(void) {
    int total=0,price[3],plan[3];
    int unit[3]={100,50,20};
    for(int i=0;i<3;i++)
        scanf("%d",&price[i]);
    for(int i=0;i<3;i++)
        scanf("%d",&plan[i]);
    for(int i=0;i<3;i++)
        total+=plan[i]/unit[i]*price[i];
    printf("%d",total);
    return 0;
}

File: 백준/Bronze/34529. Acquiring SW－IT Corn/Acquiring SW－IT Corn.java
import java.util.Scanner;
import java.util.Arrays;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int total=0,unit[]={100,50,20};
        int price[] = Arrays.stream(scan.nextLine().split(" "))
        .mapToInt(Integer::parseInt).toArray();
        int plan[] = Arrays.stream(scan.nextLine().split(" "))
        .mapToInt(Integer::parseInt).toArray();
        for(int i=0;i<3;i++)
            total+=plan[i]/unit[i]*price[i];
        System.out.println(total);
    }
}